caps/templatetags/caps_templatetags.py

```diff
@@ -41,3 +45,46 @@ def document_type(value):
         if choice[0] == value:
             return choice[1].lower()
     return "document"
+
+
+@register.tag(name="markdown")
+def markdown_tag(parser, token):
```

Comment:
I didn't notice it existed to be honest. This isn't the complicated bit so templated to keep this version rather than import another package - let me know if opposite preferred.

Reply:
No, no, can leave as it doesn't look like the other one is under super active development.